.github/workflows/ci.yml

```diff
@@ -16,6 +16,7 @@ jobs:
       - uses: actions/setup-python@v5
 
       - run: make develop
+      - run: pip install redis==4.*
```

Comment:
I feel like this shouldn't be needed -- it should pick up this dependency from setup.py already

Reply:
Without this, the `mypy` check fails

Reply:
ah yes -- redis 4.x is untyped, redis 5.x is partially typed

probably best to fix it for 5.x then?

Reply:
yes, unfortunately we can't upgrade to redis v5 yet, because `rb` fails for some reason